src/main/java/domain/user/Player.java

```diff
@@ -0,0 +1,31 @@
+package domain.user;
+
+import domain.money.BettingMoney;
+
+import java.util.Objects;
+
+public class Player extends Participant {
+
+    private final BettingMoney bettingMoney;
+    private final Name name;
+
+    public Player(final Name name, final BettingMoney bettingMoney) {
+        Objects.requireNonNull(name, "null 비허용");
+        Objects.requireNonNull(bettingMoney, "null 비허용");
+        this.name = name;
+        this.bettingMoney = bettingMoney;
```

Comment:
```suggestion
        this.name = Objects.requireNonNull(name, "null 비허용");
        this.bettingMoney = Objects.requireNonNull(bettingMoney, "null 비허용");
```

취향 차이지만 추가적으로 검증해야 할 로직이 없다면 이렇게 해봐도 괜찮을 것 같아요.